source/extensions/filters/http/aws_lambda/request_response.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+// The structures are used for the purpose of JSON (de)serialization.
+package source.extensions.filters.http.aws_lambda;
+
+import "validate/validate.proto";
+
+message Request {
+  string path = 1 [(validate.rules).string = {min_len: 1}];
+  string http_method = 2 [(validate.rules).string = {min_len: 1}];
+  // multi-value headers are coalesced with a comma separator
+  map<string, string> headers = 3;
+  // multi-value keys are overwritten. Last one wins.
+  map<string, string> query_string_parameters = 4;
+  bool is_base64_encoded = 5;
+  string body = 6;
+}
+
+// camelCase naming convention is used here because:
```

Comment:
protobuf's JSON format is supposed to automatically convert between `lower_snake_case` and `lowerCamelCase`